client/App.tsx

```diff
@@ -1,65 +1,163 @@
+    const toRotate = ['GraphQL Gateway developer portal'];
+    const period = 500;
+    const [isCopied, setIsCopied] = useState(false);
+
+    const tick = () => {
+        const i = loopNum % toRotate.length;
+        const fullText = toRotate[i];
+        const updatedText = isDeleting
+            ? fullText.substring(0, text.length - 1)
+            : fullText.substring(0, text.length + 1);
+
+        setText(updatedText);
+
+        if (isDeleting) {
+            setDelta((prevDelta) => prevDelta / 2);
+        }
+
+        if (!isDeleting && updatedText === fullText) {
+            setIsDeleting(true);
+            setIndex((prevIndex) => prevIndex - 1);
+            setDelta(period);
+        } else if (isDeleting && updatedText === '') {
+            setIsDeleting(false);
+            setLoopNum(loopNum + 1);
+            setIndex(1);
+            setDelta(500);
+        } else {
+            setIndex((prevIndex) => prevIndex + 1);
+        }
+    };
+
+    useEffect(() => {
+        const ticker = setInterval(() => {
+            tick();
+        }, delta);
+
+        return () => {
+            clearInterval(ticker);
+        };
+    }, [text]);
+
+    const code = `
+    // import package
+    import gateLogger from 'gate-logger';
+    import expressGraphQLRateLimiter from 'graphqlgate';
+
+    /**
+     * Import other dependencies
+     * */
+
+    //Add the logger middleware into your GraphQL middleware chain
+    app.use('gql', gateLogger(/*PROJECT ID*/, /*API KEY*/));
+
+    //Add the rate limiting middleware
+    app.use(
+        'gql',
+        expressGraphQLRateLimiter(schemaObject, {
+            rateLimiter: {
+                type: 'TOKEN_BUCKET',
+                refillRate: 10,
+                capacity: 100,
+            },
+        }) /** add GraphQL server here*/
+    )
+    `;
+
+    function copy(someText: string) {
+        navigator.clipboard.writeText(someText);
+        setIsCopied(true);
+    }
+    useEffect(() => {
+        setTimeout(() => {
+            if (isCopied) setIsCopied(false);
+        }, 2000);
+    });
+
     return (
         <div>
             <div className="container">
                 <section id="pageOne">
-                    <div className="blue-filter" />
-                    <div className="video_wrapper">
-                        <iframe
-                            title="unique-title-1"
-                            className="modest-vid-embed__item"
-                            style={{ width: '100%', height: '130%' }}
-                            src="https://www.youtube.com/embed/BQ0mxQXmLsk?mute=1&autoplay=1&modestbranding=1&loop=1&rel=0&amp;controls=0&amp;showinfo=0&playlist=rwlzvGfGXn4"
-                            frameBorder="0"
-                            allowFullScreen
-                        />
-                    </div>
                     <div id="pageOneTextBox">
-                        <h1>GraphQLGate</h1>
+                        <h1>GraphQL Gate</h1>
```

Comment:
I'm thinking one word, `GraphQLGate`, is more consistent with the writing everywhere else. Thoughts on this?

Reply:
Agreed